### PR TITLE
Filter count

### PR DESCRIPTION
**Pull request by emilianoyaryura**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

## -

- **How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by adamsoffer** [comment]
Out of curiosity, what are you using `tableId` for? Should that match the `queryKey`?

File: packages/www/components/Dashboard/AllSessionsTable/index.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ type SessionsTableData = {
 const AllSessionsTable = ({ title = "Sessions" }: { title?: string }) => {
   const { user, getStreamSessionsByUserId, deleteStream, deleteStreams } =
     useApi();
-  const tableProps = useTableState({ pageSize: 50 });
+  const tableProps = useTableState({ pageSize: 50, tableId: "sessions" });
```
Comment (adamsoffer) [inline packages/www/components/Dashboard/AllSessionsTable/index.tsx]: Out of curiosity, what are you using `tableId` for? I noticed its different than `queryKey`.

File: packages/www/components/Dashboard/SessionsTable/index.tsx [hunk 2]
```diff
@@ -96,7 +96,10 @@ const StreamSessionsTable = ({
   mt?: string | number;
 }) => {
   const { user, getStreamSessions } = useApi();
-  const tableProps = useTableState({ pageSize: 50 });
+  const tableProps = useTableState({
+    pageSize: 50,
```
Comment (adamsoffer) [inline packages/www/components/Dashboard/SessionsTable/index.tsx]: Our table page size is 20 per row. Should this default be set to 20?

File: packages/www/components/Dashboard/TokenTable/index.tsx [hunk 3]
```diff
@@ -52,7 +52,7 @@ const TokenTable = ({
   userId: string;
 }) => {
   const { getApiTokens, deleteApiToken } = useApi();
-  const tableProps = useTableState({ pageSize: 30 });
+  const tableProps = useTableState({ tableId: "TokenTable", pageSize: 30 });
```
Comment (adamsoffer) [inline packages/www/components/Dashboard/TokenTable/index.tsx]: Default `pageSize` should be 20.

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: packages/www/components/Dashboard/TokenTable/index.tsx (hunk 3 above)
Comment (julianbenegas) [reply packages/www/components/Dashboard/TokenTable/index.tsx]: done

File: packages/www/components/Dashboard/SessionsTable/index.tsx (hunk 2 above)
Comment (julianbenegas) [reply packages/www/components/Dashboard/SessionsTable/index.tsx]: done

File: packages/www/components/Dashboard/AllSessionsTable/index.tsx (hunk 1 above)
Comment (julianbenegas) [reply packages/www/components/Dashboard/AllSessionsTable/index.tsx]: we're using `tableId`. It was wrongly used in this PR. I fixed it now. `queryKey` is unnecessary, so I removed it

**Comment by adamsoffer** [comment]
looks good thanks!

**Pull request merged**